backend/report_submission/templates/report_submission/gen-form.html

```diff
@@ -134,7 +134,9 @@
                                    type="text"
                                    id="audit_period_other_months"
                                    name="audit_period_other_months"
-                                   disabled/>
+                                   value="{{ audit_period_other_months | default_if_none:'' }}"
+                                   {% if audit_period_other_months.len == 0 %} disabled {% endif %}
+                                   />
```

Comment:
This change adds an empty string as a default value and only disables the form if that value is empty. I _think_ the always-on `disabled` state was clobbering the input value before.